### DISCUSSION TITLE
Arduino IDE build for M5Stickc

### DISCUSSION THREAD

**Discussion by tcahall** [josephdadams/TallyArbiter#714] [Q&A]
I am trying to compile the listener for an M5stick and getting a missing SensorDataStruc.h.  Web searches haven't located the missing file.  Error below.
In file included from /Users/timothycahall/Downloads/m5stickc-listener/utility/MPU6886.h:13,
                 from /Users/timothycahall/Downloads/m5stickc-listener/M5StickC.h:84,
                 from /Users/timothycahall/Downloads/m5stickc-listener/m5stickc-listener.ino:13:
/Users/timothycahall/Downloads/m5stickc-listener/utility/MahonyAHRS.h:18:10: fatal error: SensorDataStruct.h: No such file or directory
   18 | #include "SensorDataStruct.h"
      |          ^~~~~~~~~~~~~~~~~~~~
compilation terminated.

exit status 1

Compilation error: SensorDataStruct.h: No such file or directory

**Comment by Jocke4f** [comment]
Are all libraries listed in https://github.com/josephdadams/TallyArbiter/blob/master/listener_clients/m5stickc-listener/README.md included?
Which version of Arduino IDE are you using? I compiled it successfully on v2.3.2 on Windows 10.
